crates/iota/src/key_identity.rs

```diff
@@ -41,19 +47,33 @@ impl Display for KeyIdentity {
         let v = match self {
             KeyIdentity::Address(x) => x.to_string(),
             KeyIdentity::Alias(x) => x.to_string(),
+            #[cfg(feature = "iota-names")]
+            KeyIdentity::Name(x) => x.to_string(),
         };
         write!(f, "{}", v)
     }
 }
 
 /// Get the IotaAddress corresponding to this key identity.
 /// If no string is provided, then the current active address is returned.
-pub fn get_identity_address(
+pub async fn get_identity_address(
     input: Option<KeyIdentity>,
-    ctx: &mut WalletContext,
+    ctx: &WalletContext,
 ) -> Result<IotaAddress, Error> {
     if let Some(addr) = input {
-        get_identity_address_from_keystore(addr, ctx.config().keystore())
```

Comment:
Why are we not replacing the calls to `get_identity_address_from_keystore` with `get_identity_address`?